array_reverse/array-reverse.js

```diff
@@ -0,0 +1,13 @@
+const arrOne = [1, 2, 3, 4, 5, 6];
+
```

Comment:
This can be deleted from production code, since you're just writing the function itself.